netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+  publish = "_site"
+  command = "npm run build"
```

Comment:
I have another question @calebeby. :)

Do we need to do anything with the Netlify "Continuous Deployment" settings? Or does this `netlify.toml` override these settings?

![screen shot 2018-07-11 at 11 08 56 am](https://user-images.githubusercontent.com/459757/42591286-24888432-84fb-11e8-8283-623e8adecb9d.png)

Reply:
netlify.toml will override the web config

Generally I like using netlify.toml over the web configuration because then you can settings changes in PRs without messing up other branches

Reply:
@calebeby Cool, works for me! I just wanted to make sure I understood. Thanks for the explanation! 😉